docs/BasicCommunication/OnAppDeactivated/index.md

```diff
@@ -11,19 +11,15 @@ Purpose
 
 SDL requires this notification in order to keep the mobile application from sending RPC's related to the HMI's functionality (e.g. adding commands for VR, starting an interaction with the user, speaking text via TTS, etc).
 
-!!! MUST
-
-1. Send an OnAppDeactivated notification when the user has navigated away from the application because of some `DeactivateReason`
-2. Send an OnAppDeactivated (reason="NAVIGATIONMAP") notification when the user switches from the application layout to the embedded navigation screen.
-3. Send an OnAppDeactivated (reason="AUDIO") notification when the user switches to the embedded audio source.
-4. Send an OnAppDeactivated (reason="PHONEMENU") notification when the user switches to the HMI phone menu.
-
-!!!
 
 In the event a navigation application is in `FULL` or `LIMITED` and the HMI sends `OnAppDeactivated` (reason="NAVIGATIONMAP"), SDL must set the navigation application to `BACKGROUND`.
```

Comment:
still has 
`OnAppDeactivated` (reason="NAVIGATIONMAP")

Reply:
fixed in 0a9d1ea